views/route_segments/route_segments_show.html

```diff
@@ -28,7 +29,7 @@ <h2 class="{{ IconFor `route-segment` }}">{{ .Name }}</h2>
           <div class="inner-form">{{ template "route_segment_details" . }}</div>
         </div>
         <div class="basis-1/2 2xl:basis-1/3">
-          <div class="inner-form">
+          <div class="inner-form" style="max-height: 640px; overflow-y: auto">
```

Comment:
With tailwind: class="inner-form max-h-[640px] overflow-y-auto"

Reply:
we should probably add the `max-h-[640px]` to the breakdown in `views/workouts/workouts_show.html`

Reply:
Sorry I am completely unfamiliar with Tailwind (I am quite experienced with CSS and I find Tailwind just... odd syntax sugar, but people seem to like it). Will copy your suggestion!

Reply:
I'm not bad at CSS, but I used CSS like 15 years ago; so much has changed that I have no clue how to perform things; this is where Tailwind comes in handy: I just search their site for what I want to achieve, and they named the "commands" sanely... Want to center text (you know, the hardest thing in CSS)? Search for "center text". Most commands are indeed just the css key+value, but often they are more sane.